ui/components/app/hold-to-reveal-button/hold-to-reveal-button.js

```diff
@@ -93,6 +121,7 @@ export default function HoldToRevealButton({ buttonText, onLongPressed }) {
         <Box className="hold-to-reveal-button__absolute-fill">
           <svg className="hold-to-reveal-button__circle-svg">
             <circle
+              aria-label="circle-locked"
```

Comment:
Are these meant to be in this PR?  They don't seem related to metrics.

Reply:
yes, these are for the tests to trigger the unlock